velox/docs/monthly-updates/march-2022.rst

```diff
@@ -3,12 +3,12 @@ March 2022 Update
 ********************
 
 Documentation
--------------
+=============
```

Comment:
I wish there was a way to keep this formatting, but change Feature or the Month formatting. This way the formatting would be consistent across all updates. Not a big deal though. Thanks for fixing.

Reply:
using "=============" as in my current change is the right thing. The existing "-------------" works because Sphinx is smart and it can infer automatically some cases. More details if you are curious https://rest-sphinx-memo.readthedocs.io/en/latest/ReST.html#sectioning